python/tests/scripts/b0_chaining_prelim.metta

```diff
@@ -1,35 +1,42 @@
-; From matching to rule chaining
-
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+; From matching to rule chaining:
```

Comment:
Can we replace "rule" with something else here? I'm ok with it, but it sounds not perfect, because these are not necessarily rules. We are talking about either equalities, which matching is chained by the interpreter, or or just arbitrary pattern matching queries. Maybe, we can more verbose here, e.g. "From matching to evaluation of expressions via equality queries chaining" or something like this.